ui/src/Groups/Counter/Counter.view.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/no-multi-comp */ // disabled as we have a hoc funtion in file
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Body, BODY, Subheading, Text } from '../../Elements/Text/index.js';
+
+const Counter = (props) => {
+  const { title, subtitle, count, countLimit, className, ...others } = props;
+  const classesToApply = `Counter${className ? ` ${className}` : ''}`;
+
+  let countValue = count;
+  if (count < 0) {
+    countValue = '00';
+  } else if (count < 10) {
+    countValue = '0' + count;
+  } else if (count > countLimit) {
+    countValue = countLimit + '+';
+  }
+
+  return (
+    <div {...others} className={classesToApply}>
+      <div className={'Counter--title'}>
+        <Subheading>{title}</Subheading>
+      </div>
+      <div className={'Counter--subtitle'}>
+        <Body>{subtitle}</Body>
+      </div>
+      <div className={'Counter--count'}>
+        <Text type={BODY} className={'Counter--countValue'}>
```

Comment:
Were you not able to use a `Body` component with a className prop?

Reply:
I wasn't sure if that was appropriate here as I was immediately overriding the styling with my own. Have changed to a `<Body>`.